master/getting-started/kubernetes/installation/hosted/hosted.md

```diff
@@ -3,15 +3,19 @@ title: Standard Hosted Install
 canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/hosted'
 ---
 
-The following steps install {{site.prodname}} as a Kubernetes add-on using your own etcd cluster.
+The following steps install {{site.prodname}} as a Kubernetes add-on using 
```

Comment:
As an FYI, it's fine to do this for new text but we don't have a requirement to limit to 79 characters or whatever, and in cases like this where it's not really much longer we strongly prefer to leave it as is in order to minimize the size of the diff.

Modern screen widths allow for for viewing slightly longer lines.